src/navigator/types.tsx

```diff
@@ -81,11 +81,11 @@ export type StackParamList = {
   [Screens.EarnInfoScreen]: undefined
   [Screens.EarnEnterAmount]: {
     pool: EarnPosition
-    mode?: EarnEnterMode
+    mode?: EarnActiveAction
```

Comment:
same comment as above, the screen shouldn't need to deal with invalid action names.

Reply:
Narrowed in ff2bd43cf3f9794d2b053f71d3ea04e459f2698f!